paper_framework_sources/p1898_forward_progress_delegation.md

```diff
@@ -301,76 +311,61 @@ When applied to a `receiver`, if supported, will return a `scheduler` that the `
 
 ### Wording
 The name `execution::get_scheduler` denotes a customization point object.
-The expression `execution::get_scheduler(R)` for some subexpression `R` is expression-equivalent to:
+For some subexpression `r`, let `R` be a type such that `decltype((r))` is `R`
+The expression `execution::get_scheduler(r)` is expression-equivalent to:
 
- * `R.get_scheduler()` if that expression is valid.
- * Otherwise, `get_scheduler(R)` if that expression is valid with overload resolution performed in a context that includes the declaration
+ * `r.get_scheduler()` if that expression is valid, if `R` satisfies `receiver`
+ * Otherwise, `get_scheduler(r)` if that expression is valid, and if `R` satisfies `receiver`, with overload resolution performed in a context that includes the declaration
  ```
```

Comment:
get_scheduler should work on more things than receiver. Perhaps change receiver to scheduler_provider?

Reply:
True enough. Changing.